docs/virtual_machines/disks_and_volumes.md

```diff
@@ -1403,68 +1403,217 @@ Test awesome shareable disks150+0 records in
 
 If you are using local devices or RWO PVCs, setting the affinity on the VMs that share the storage guarantees they will be scheduled on the same node. In the example, we set the affinity on the second VM using the label used on the first VM. If you are using shared storage with RWX PVCs, then the affinity rule is not necessary as the storage can be attached simultaneously on multiple nodes.
 
-## File Systems
+## Sharing Directories with VMs
 
-Currently, only `virtIO-FS` file system is supported.
-### virtIO-FS
-
-`virtIO-FS` makes a Persistent Volume Claim visible in the `kubevirt` VM.
+`Virtiofs` allows to make visible external filesystems to `KubeVirt` VMs.
+`Virtiofs` is a shared file system that lets VMs access a directory tree on the host.
+Further details can be found at [Official Virtiofs Site](https://virtio-fs.gitlab.io/).
 
 > **Note:** The **ExperimentalVirtiofsSupport** feature gate
 > [must be enabled](../operations/activating_feature_gates.md#how-to-activate-a-feature-gate)
-> to use this volume. This feature is available starting with KubeVirt v0.36.0.
+> to share directories with VMs. This feature is available starting with KubeVirt v0.36.0.
+
+### Sharing Persistent Volume Claims
+#### Cluster Configuration
 
+We need to create a new VM definition including the `spec.devices.disk.filesystems.virtiofs` and a PVC.
 Example:
 
 ```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  name: testvmi-fs
+spec:
+  domain:
+    devices:
+      disks:
+        - disk:
+            bus: virtio
+          name: containerdisk
+        - disk:
+            bus: virtio
+          name: cloudinitdisk
+      filesystems:
+        - name: virtiofs-disk
+          virtiofs: {}
+    resources:
+      requests:
+        memory: 1024Mi
+  volumes:
+    - name: containerdisk
+      containerDisk:
+        image: quay.io/containerdisks/fedora:latest
+    - cloudInitNoCloud:
+        userData: |-
+          #cloud-config
+          password: fedora
+          chpasswd: { expire: False }
+      name: cloudinitdisk
+    - name: virtiofs-disk
+      persistentVolumeClaim:
+        claimName: mypvc
+```
+
+#### Configuration Inside the VM
+The following configuration can be done in using startup script. See [cloudInitNoCloud](#cloudinitnocloud) section for 
+more details. 
+However, we can do it manually by login inside the VM and mounting it. 
```

Comment:
nit
```suggestion
However, we can do it manually by logging into the VM and mounting it. 
```

Reply:
To nit your nit, it should be "...by logging in to the VM..."
('Logging in' or 'log in' is a phrasal verb and 'to' is the preposition that connects it to the object.)

Reply:
Thanks for the clarification. Done.